Lib/test/_test_multiprocessing.py

```diff
@@ -3929,8 +3926,11 @@ def test_shared_memory_ShareableList_pickling(self):
         sl.shm.close()
 
     def test_shared_memory_cleaned_after_process_termination(self):
-        import subprocess
-        from multiprocessing import shared_memory
+        # NOTE: this test emits a UserWarning
```

Comment:
adding `stderr=subprocess.PIPE` in the `subprocess.Popen` call does the job locally for me.

Reply:
Ahah. Will try.